Squirrel/Program.cs

```diff
@@ -16,7 +17,15 @@
 
 services.AddTransient<IEmailSender, EmailSender>();
 
-services.AddAuthentication().AddCookie(CookieAuthenticationDefaults.AuthenticationScheme);
+services.AddAuthentication()
```

Comment:
Could be boxed into an extension method to keep `Program.cs` cleaner. For instance 
`public static IServiceCollection AddGoogleAuthWithCookies(this IServiceCollection services)`.
What do you think?